core/spec/models/spree/order_cancellations_spec.rb

```diff
@@ -152,7 +152,9 @@
 
       it "generates the correct total amount" do
         order.cancellations.short_ship([inventory_unit_1])
+        inventory_unit_2.line_item.reload # UnitCancel#compute_amount needs updated amounts
         order.cancellations.short_ship([inventory_unit_2])
+        line_item.reload
```

Comment:
@jhawthorn great point.  Here's a PR (against this PR) to do that: https://github.com/jordan-brough/solidus/pull/7  Shall I merge that into this PR?